input-forms.md

```diff
@@ -0,0 +1,137 @@
+# Input-forms Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/configuration/input-forms**   
+
+Provide access to the configured input-forms. It returns the list of existent input-forms.
+
+Example: to be provided
+
+**/api/configuration/value-pairs**   
```

Comment:
Do we need a separate `value-pairs` endpoint? Can't we embed them in a field definition of the `/api/configuration/input-forms/<:form-name>` endpoint? This would make the logic on (any future) front-end side easier.

Reply:
The value-pairs already have an identifier that make them candidate to be REST resources. I'm aware of customization where other step than the DescribeStep, the one that use the inputform, also get options for some dropdowns from the input-form value-pairs section. Moreover the same value pair can be reused by several configuration so it will be easier to just link the same resource instead to create a different copy.
Please note that the client, thanks to the embedding feature of HAL, is not required to fire a separate call to retrieve the needed value-pairs